wagtail/contrib/forms/forms.py

```diff
@@ -47,25 +61,19 @@ def create_number_field(self, field, options):
         return django.forms.DecimalField(**options)
 
     def create_dropdown_field(self, field, options):
-        options["choices"] = map(
-            lambda x: (x.strip(), x.strip()), field.choices.split(",")
-        )
+        options["choices"] = self.format_choices(field)
         return django.forms.ChoiceField(**options)
 
     def create_multiselect_field(self, field, options):
-        options["choices"] = map(
-            lambda x: (x.strip(), x.strip()), field.choices.split(",")
-        )
+        options["choices"] = self.format_choices(field)
         return django.forms.MultipleChoiceField(**options)
 
     def create_radio_field(self, field, options):
-        options["choices"] = map(
-            lambda x: (x.strip(), x.strip()), field.choices.split(",")
-        )
+        options["choices"] = self.format_choices(field)
         return django.forms.ChoiceField(widget=django.forms.RadioSelect, **options)
 
     def create_checkboxes_field(self, field, options):
-        options["choices"] = [(x.strip(), x.strip()) for x in field.choices.split(",")]
+        options["choices"] = self.format_choices(field)
         options["initial"] = [x.strip() for x in field.default_value.split(",")]
```

Comment:
We will likely need to update this initial value to also use the same function so that it handles new lines.

Reply:
@lb- May need to use a new function since the initial value requires list of values rather than list of key and value pair.

Reply:
could use the same function and then just get the values from the dict result maybe?

Reply:
Okay will try that.